apprise/Apprise.py

```diff
@@ -124,7 +122,7 @@ def instantiate(url, asset=None, tag=None, suppress_exceptions=True):
         # Prepare our Asset Object
         asset = asset if isinstance(asset, AppriseAsset) else AppriseAsset()
 
-        if isinstance(url, six.string_types):
+        if isinstance(url, str):
```

Comment:
On a recent run, I can observe that `six` is still being installed. Do you have a clue why?

-- https://app.travis-ci.com/github/caronc/apprise/jobs/585073979#L363

Reply:
I see, `pipdeptree` is telling me that `six` is a dependency of `tox`.

Reply:
Yeah, that's okay though, `six` is still used for so many packages that strive to continue to be backwards compatible for the audiences.  It won't be pulled into production anymore at least (still one less package; so that's good).